app/assets/stylesheets/_navbar.scss

```diff
@@ -181,3 +146,35 @@
     margin-right: 0;
   }
 }
+
+.locale {
```

Comment:
I think this name is too generic, can we make it clearer for who reads the code? (It's not a blocker so won't add it to the review, but I think we could rename it, what do you think?)